### ISSUE TITLE
Can you support AndroidX?

### ISSUE THREAD

**Issue by 0xByteLeon** [kayvannj/PermissionUtil#16]
Android support package name already changed

**Comment by kayvannj** [comment]
yes, I started a new version of this lib that will use the AndroidX and is all Kotlin. I'll have it pushed up in a few days.

**Comment by levibostian** [comment]
I would be happy to contribute to the AndroidX and Kotlin version, @kayvannj if you need the help.

**Comment by kayvannj** [comment]
Hey @levibostian, I haven't gotten around to it but yes it would be greatly appreciated if you can contribute. Feel free to push up a PR and I'll review it.

> I would be happy to contribute to the AndroidX and Kotlin version, @kayvannj if you need the help.

**Comment by levibostian** [comment]
Is there a specific branch you have begun work on that I can continue with? Or would I need to start from scratch?

**Comment by kayvannj** [comment]
My first stab at it is in this branch https://github.com/kayvannj/PermissionUtil/commits/kotlin-wip
But I don't recall if it's finished up or not. the switch to kotlin and android-x can be a major version so don't worry about breaking the API.
